packages/manager/modules/overthebox/src/overthebox/autoconfigure/dns/template.html

```diff
@@ -0,0 +1,247 @@
+<div>
+    <oui-message
+        data-ng-if="$ctrl.successMessage"
+        type="success"
+        aria-close-button-label="{{:: 'overTheBox_autoconfigure_close' | translate}}"
+        dismissable
+    >
+        <span data-ng-bind="$ctrl.successMessage"></span>
+    </oui-message>
+    <oui-message
+        data-ng-if="$ctrl.errorMessage"
+        type="error"
+        aria-close-button-label="{{:: 'overTheBox_autoconfigure_close' | translate}}"
+        dismissable
+    >
+        <span data-ng-bind="$ctrl.errorMessage"></span>
+    </oui-message>
+    <div class="oui-box oui-box_light">
+        <h4
+            class="oui-box__heading"
+            data-translate="overTheBox_autoconfigure_dns_server"
+        ></h4>
+        <div data-ng-if="$ctrl.hasNoNameServer()" class="mb-2">
+            <em data-translate="overTheBox_autoconfigure_no_values"></em>
+        </div>
+        <form novalidate name="formServer" data-ng-if="$ctrl.showAddServer">
+            <div class="oui-input-group oui-input-group_inline">
+                <oui-field
+                    label="{{:: 'overTheBox_autoconfigure_dns_server' | translate}}"
+                    label-popover="{{:: 'overTheBox_autoconfigure_dns_server_description' | translate }}"
+                >
+                    <input
+                        class="oui-input"
+                        name="serverName"
+                        required
+                        ng-model="$ctrl.nameserver.server"
+                    />
+                </oui-field>
+                <oui-field
+                    class="ml-2"
+                    label="{{:: 'overTheBox_autoconfigure_priority' | translate}}"
+                >
+                    <input
+                        class="oui-input"
+                        name="serverPriority"
+                        type="number"
+                        required
+                        ng-model="$ctrl.nameserver.priority"
```

Comment:
We must to have the `data` for no native HTML elements attributes.
Else we lost the HTML validation page.
```suggestion
                        data-ng-model="$ctrl.nameserver.priority"
```